### ISSUE TITLE
Allow Users to Enter Standard Curves

### ISSUE THREAD

**Issue by Luke-Malchow-Bergenthal** [UPDDI/mps-database-server#364]
It has been requested that users be permitted to enter standard curves into the database. 

At the moment it is technically possible to submit standard curves in the supplementary data field for a study.

Where precisely and how to treat these curves is something I can not say off the top of my head. What's more, whether or not these curves can be feasibly used to process data is up to debate (I am not able to say how standard curves may or may not change from assay to assay).

**Comment by Luke-Malchow-Bergenthal** [comment]
Note that it is possible in plates to specify wells as being for a "standard." However at the moment it is not possible to indicate the dilution series used for the standard. This would have to change before we can meaningfully add standard data.

**Comment by Luke-Malchow-Bergenthal** [comment]
It is unclear whether we are still interested in storing this data in the database itself (that is to say, in a table rather than in an associated file). If we decide definitively that we are not interested in storing such elements of raw data, this issue should be closed.

**Comment by ahgough** [comment]
Correct, it is not clear whether we will support standard curves. I would leave this in Github, but marked something like Question/decision.

Bert
